src/templates/java/BaseNode.java.ftl

```diff
@@ -285,4 +285,10 @@ public class ${grammar.baseNodeClassName} implements Node {
         }
         return buf.toString();
     }
+    
+    @Override
```

Comment:
Using @Override here is only valuable for Java >=1.6. Is it a problem? If no, I think other methods should use @Override.

Reply:
I think my intention now is only to support Java >=1.8. So, no, no problem at all.

Reply:
Ok thanks for the info. If you like it, please merge it.

Reply:
I merged it, but I should comment that I hope you understand that there is no absolute need to do a PR like this to add a method to the relevant file. I mean, if you want Token to have an extra method (or a few extra ones) you can just use INJECT(Token)... or it could be to insert a method into Token or BaseNode or wherever you want. Of course, there is a difference, which is that the method is only available in YOUR project, whereas, this way, the method we just added is available in EVERY project. 

Probably, moving forward, it is better to start with just adding the methods you need in your own project, but if the case is pretty clear that this is just something generally useful, then by all means, we just add it to the template so it is generally available. Of course, there is a tradeoff between adding a HUGE number of potentially useful methods and keeping a lean base API and expecting the user to just add the additional methods he needs. Of course, if the core API is very well designed, it should be easy for the user to add a few methods to get the exact functionality he needs. But, you know, this is all in flux right now anyway, so we can experiment a bit before saying any of this is final. But that situation changes once there are thousands of end users!

Reply:
I understand what you mean and i agree with you. We must provide only relevant methods. It's m'y goal to provide the same api than W3c node with find methods. We need tests for those methods. Perhaps WE could use json parser. What do you think about that?

Reply:
Sure. I'm totally in favor of having tests. And the json parser is an okay place because that is a very small, simple grammar. I actually copied the json example over to the javacc21 repository and added it as a test. For now, you can feel free to do whatever you want in the blue-sky repository. It exists just for experimentation really.